x-pack/plugins/apm/public/services/rest/apm_observability_overview_fetchers.ts

```diff
@@ -42,17 +41,12 @@ export const fetchObservabilityOverviewPageData = async ({
       },
       transactions: {
         type: 'number',
-        value:
-          mean(
-            transactionCoordinates
-              .map(({ y }) => y)
-              .filter((y) => y && isFinite(y))
-          ) || 0,
+        value: transactionPerMinute.value || 0,
       },
     },
     series: {
       transactions: {
-        coordinates: transactionCoordinates,
+        coordinates: transactionPerMinute.timeseries,
```

Comment:
Can we call this `timeseries` as well. Or is that the api contract?

```suggestion
        timeseries: transactionPerMinute.timeseries,
```

Reply:
It's the API contract.